.editorconfig

```diff
@@ -0,0 +1,20 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
```

Comment:
Should be "tab". The only file that uses spaces for indentation appears to be lib/Utils.php (it actually mixes the two).